packages/driver/src/multi-domain/logs.ts

```diff
@@ -4,11 +4,11 @@ import $Log from '../cypress/log'
 
 export const handleLogs = (Cypress: Cypress.Cypress, specBridgeCommunicator: SpecBridgeDomainCommunicator) => {
   const onLogAdded = (attrs) => {
-    specBridgeCommunicator.toPrimary('log:added', attrs, $Log.toSerializedJSON)
+    specBridgeCommunicator.toPrimary('log:added', $Log.toSerializedJSON(attrs))
```

Comment:
I opted to serialize here instead of creating a specific 'to primary' function because the serialization is so simple.

Reply:
and confined to one file